web/source/osk/banner.ts

```diff
@@ -390,8 +390,28 @@ namespace com.keyman.osk {
       keyman.core.languageProcessor.addListener('suggestionsready', manager.updateSuggestions);
       keyman.core.languageProcessor.addListener('tryaccept', manager.tryAccept);
       keyman.core.languageProcessor.addListener('tryrevert', manager.tryRevert);
+      keyman.core.languageProcessor.addListener('suggestionapplied', this.suggestionApplied);
     }
 
+    /**
+     * Handler for post-porcessing once a suggestion has been applied: calls
```

Comment:
```suggestion
     * Handler for post-processing once a suggestion has been applied: calls
```

Reply:
I left this one there to see if anyone would pick it up.

Just kidding. Good catch 🐟